packages/iov-cosmos/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@iov/cosmos",
+  "version": "1.0.0-alpha.1",
+  "description": "Transaction codec and client to communicate with the Cosmos Hub blockchain",
+  "author": "IOV SAS <admin@iov.one>",
+  "license": "Apache-2.0",
+  "main": "build/index.js",
+  "types": "types/index.d.ts",
+  "files": [
+    "build/",
+    "types/",
+    "*.md",
+    "!*.spec.*",
+    "!**/testdata/"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iov-one/iov-core/tree/master/packages/iov-cosmos"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
Can you set this to private (which should ensure it is not published)?

Reply:
I do not yet understand the difference between this setting and the main `"private": true`. See https://github.com/lerna/lerna/issues/1942#issuecomment-468001574

Maybe you can have a look into this

Reply:
The only valid values for `publishConfig.access` are `public` and `restricted`, neither of which prevents publishing. What we want is `private` set to `true`.